app/views/profile/index.html.slim

```diff
@@ -1,5 +1,4 @@
-- content_for(:nav) do
-  = render 'shared/nav_auth'
+= shared_nav(greeting: @view_model.header_personalization)
```

Comment:
@jessieay I added a spec for the `header_personalization` method. I will put in a follow up PR with tests for the rest of the ProfileIndex methods (looks like we missed adding a spec file for it 😬 )

What do you think about moving the `nav_auth` file from `shared` into `profile`? It will require moving a few other things around (mostly references to the 'shared.nav_auth_welcome' translation) but may be worth it since this partial is only used on the profile page.